src/components/Sections/SectionView.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react'
+import cx from 'classnames'
+
+import useBreakpoints from 'cozy-ui/transpiled/react/providers/Breakpoints'
+
+import {
+  DisplayMode,
+  Section,
+  SectionViewProps
+} from 'components/Sections/SectionsTypes'
+import { SectionHeader } from 'components/Sections/SectionHeader'
+import { ShortcutLink } from 'components/ShortcutLink'
+
+const computeDisplayMode = (
+  isMobile: boolean,
+  section: Section
+): DisplayMode => {
+  const layout = section.layout[isMobile ? 'mobile' : 'desktop']
+  return layout.detailedLine ? DisplayMode.DETAILED : DisplayMode.COMPACT
+}
+
+export const SectionView = ({ section }: SectionViewProps): JSX.Element => {
+  const [menuState, setMenuState] = useState(false)
```

Comment:
nit: I think `menuState`, `anchorRef`, `toogleMenu` are only used into `SectionHeader` so they can be regroup there as they are only related to the more button

Reply:
I'm not sure, I think the whole section should be aware of the menu state, I'll reassess when implementing it